README.md

```
@@ -60,6 +82,37 @@ Handles file system operations and provides a secure bridge between the frontend
+
+<div align="center">
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="docs/assets/architectual-design-dark.svg">
+    <source media="(prefers-color-scheme: light)" srcset="docs/assets/architectual-design-light.svg">
+    <img src="docs/assets/architectual-design-light.svg" alt="PictoPy architecture diagram">
+  </picture>
+</div>
+
+## Want to Contribute?
+
+<a href="https://discord.gg/hjUhu33uAn"><img src="https://img.shields.io/discord/1022871757289422898?style=flat&logo=discord&logoColor=white&logoSize=auto&label=Discord&labelColor=5865F2&color=57F287" height="30"></a>
```

markdownlint-cli2 (0.22.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)